packages/databricks-vscode/src/quickstart/QuickstartCommands.ts

```diff
@@ -6,7 +6,7 @@ export class QuickstartCommands {
     openQuickstartCommand() {
         return async () => {
             const uri = Uri.file(
-                this.context.asAbsolutePath("README.quickstart.md")
+                this.context.asAbsolutePath("DATABRICKS.quickstart.md")
```

Comment:
The file is referenced  in the `README.md`, which needs to be updated as well.

What was the reason was renamed?